### ISSUE TITLE
Fonts are not displayed correctly in SVG

### ISSUE THREAD

**Issue by jourmager** [sumatrapdfreader/sumatrapdf#3747]
Fonts are not displayed correctly in SVG files
Windows 10
Left - SumatraPDF 3.5.15703 64-bit portable pre-release (or 3.4.6)
Right - Google Chrome 117 (or Edge 117)
![svg](https://github.com/sumatrapdfreader/sumatrapdf/assets/76526957/7d0a305a-e12e-474c-8a27-6ebdab28edb7)
[Original file](https://upload.wikimedia.org/wikipedia/commons/0/00/Solar_System_size_to_scale.svg)

**Comment by kimseongyu** [comment]
Hello~~!
It's an inheritance issue with the <g> element in mupdf.

**Comment by kjk** [comment]
@kimseongyu mupdf doesn't use github, doesn't merge PRs etc.

You would have to open a bug on https://bugs.ghostscript.com/ but they don't look at that much either.

The best way to get their attention is to bug them on discord channel: https://discord.gg/SnXWzqzjKs

**Comment by GitHubRulesOK** [comment]
@kjk agreed this would never be PR by mupdf (not their policy  very very rarely) but could that fix work in SumatraPDF !
